### PR TITLE
WIP: collected during relation

### PR DESCRIPTION
**Pull request by ddooley**
A relation between a material entity and an occurent in which the specimen is collected during the existence of the occurent.

### CODE REVIEW

**Comment by wdduncan** [comment]
'occurent' should be 'occurrent' (needs another 'r').

Perhaps be consistent in the use of 'material entity'? Information about 'specimen' could be in a one sentence gloss. E.g.:  
```
A relation between a material entity and an occurrent in which the material entity is collected during the existence of the occurrent. The material entity is a specimen that collected for scientific purposes.
```

**Comment by ddooley** [comment]
I made suggested change.  This can now be approved and closed!

**Review by anitacaron — DISMISSED** [review]
Technically approved. It still needs ontology approval.

**Review by anitacaron — DISMISSED** [review]
Technically approved. It still needs ontology approval.

File: src/ontology/ro-edit.owl
Comment (cmungall) [inline src/ontology/ro-edit.owl:6625]: general observation: this is occurrent, which is above the shoreline in COB. See #780. This comment is not to block the PR: clearly we have a precedent for using occurrent already

File: src/ontology/ro-edit.owl
Comment (ddooley) [reply src/ontology/ro-edit.owl:6653]: Done!

**Review by ddooley — COMMENTED** [review]
Good to go!

File: src/ontology/ro-edit.owl
Comment (anitacaron) [inline src/ontology/ro-edit.owl:47]: Why this PATO term was included in the edit file?

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/ro-edit.owl
Comment (anitacaron) [reply src/ontology/ro-edit.owl:47]: I don't think so. :) I've removed it anyway.

**Review by anitacaron — APPROVED** [review]
Technically, I approve. Still need to have an ontology review,

**Review by cmungall — APPROVED** [review]

**Pull request merged**
